README.md

```diff
@@ -31,11 +31,11 @@ class CounterContainer extends Container<CounterState> {
   };
 
   increment() {
-    this.setState({ count: this.state.count + 1 });
+    this.setState((prevState) => ({ count: prevState.count + 1 }));
```

Comment:
Cotainers are not React.Components, while they have same-ish `setState` method, it doesnt event yet allow callback-style argument - see https://github.com/thejameskyle/unstated/pull/1